server/middleware/booleanCoerce.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = key => {
+  return (req, res, next) => {
+    const value = req.body[key];
```

Comment:
boom if `req.body` is undefined

Reply:
`req.body` will always be defined because of the `body-parser` middleware.

Reply:
I added a check for this anyway.